new.js

```diff
@@ -0,0 +1,13 @@
+var country = "island Great";
+continent = "Europa";
+population = 0.039 ;
+ 
+console.log ("country-", country);
+console.log ("continent-", continent);
+console.log ("population-", population,"milion." );
+
+var island = (country=="island")
```

Comment:
I'm afraid you didn't understand the requirements.

the task was:
```Объявите четвертую переменную isIsland, чтобы указать, является ли данная страна островом или нет, присвоив данной переменной логическое значение true или false.```

so this variable should keep "true" or "false" value only.
In your case you're comparing  country name to a string "island". You will always get false unless the country name is "island"